faiss/gpu/test/TestGpuIndexCagra.cu

```diff
@@ -330,6 +330,9 @@ void copyToTest(
                 recall_score.view(),
                 copy_ref_dis_mds_opt,
                 ref_dis_mds_opt);
+        std::cout << "run: " << tries
+                  << ", recall_score: " << *recall_score.data_handle()
```

Comment:
Can we get the actual value once and re-used just to be safe?

Reply:
Assuming this is a testing artifact as well.

Reply:
Yes, testing artifact too.